factory/grpcServerFactory.go

```diff
@@ -0,0 +1,55 @@
+package factory
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/multiversx/mx-chain-ws-connector-template-go/config"
+	"github.com/multiversx/mx-chain-ws-connector-template-go/process"
+	"github.com/multiversx/mx-chain-ws-connector-template-go/server"
+)
+
+// CreateGRPCServer will create the gRPC server along with the required handlers.
+func CreateGRPCServer(
+	enableGrpcServer bool,
+	cfg config.GRPCConfig,
+	outportBlocksPool process.DataPool,
+	dataAggregator process.DataAggregator) (process.GRPCServer, process.Writer, error) {
+	if !enableGrpcServer {
+		return nil, os.Stdout, nil
```

Comment:
we can have a `disabled` component for GRPC service in case it is not enabled

Reply:
changed. but didn't implement disabled.